testng-browserstack-parallel.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we remove the .gitkeep file above.

Reply:
if we don't have file in the folder, then it will not be commited